regression-test/framework/src/main/groovy/org/apache/doris/regression/suite/Suite.groovy

```diff
@@ -2127,7 +2127,7 @@ class Suite implements GroovyInterceptable {
         def js = jsonOutput.toJson(map)
         log.info("get be metric req: ${js} ".toString())
 
-        def ret = 0;
+        def ret = Integer.MAX_VALUE
```

Comment:
I am worried that it will affect other existing test cases, so I do not recommend making this change. Is it possible to loop through some test cases in `test_be_metrics.groovy` and terminate the loop if one of them meets expectations? At least this way, you don't have to change other existing test cases.
Like this : 
```
bool haveValue = false;
for ( xxxx ){
     if (  check(xxx) ) {
         haveValue = true;
         break;
     } 
    Thread.sleep( xxx )
}

assertTrue (haveValue)

```